hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright YEAR Heptio
```

Comment:
Does this need 2018?

Reply:
It gets updated automagically, so next year it will substitute 2019 dynamically.

Reply:
I'll rename to make more clear